function ajaxRequest()
{
	//ВАЖНО!!! Данный метод не работает с синтаксическими ошибками: docment.writ("Hello")
	//1 - Выполняется код внутри блока try.
	try 
	{ 
		var request = new XMLHttpRequest();
		alert('the browser is not IE');
	}
	//2 - если выполняемом коде не произошла ошибка, то следующий код пропускаеться
	//в ином случае выполняется код в catch и в аргумент (е1) -  имя может быть любым
	//передаётся объект с ошибкой, ошибку можно вывести так: alert("Ошибка "+ e1.name + "сообщение: " + e1.message)
	//name - Тип ошибки. Например, при обращении к несуществующей переменной: "ReferenceError".
	//message - Текстовое сообщение о деталях ошибки.
    //stack - Везде, кроме IE8-, есть также свойство stack, которое содержит строку с информацией о последовательности вызовов, которая привела к ошибке.
	catch(e1)
	{
		try // Это IE 6+?
			{ // Да
			request = new ActiveXObject("Msxml2.XMLHTTP")
			}		
		catch(e2)
		{
			try // Это IE 5?
			{ // Да
			request = new ActiveXObject("Microsoft.XMLHTTP")
			}
			catch(e3) // Данный браузер не поддерживает AJAX
			{
			request = false
			}
		}
		finally {
		// Код который выполниться в любом случае, использовать необязательно
		}
	}
return request
}

//Если в нём ошибок нет, то блок catch(err) игнорируется, то есть выполнение доходит до конца try и потом прыгает через catch.
//Если в нём возникнет ошибка, то выполнение try на ней прерывается, и управление прыгает в начало блока catch(err).
//При этом переменная err (можно выбрать и другое название) будет содержать объект ошибки с подробной информацией о произошедшем.